eva.py

```diff
@@ -52,3 +51,5 @@ def eva():
 if __name__ == '__main__':
     # execute only if run as the entry point into the program
     eva()
+    mode = ConfigurationManager().get_value('core', 'mode')
+    init_builtin_udfs(mode=mode)
```

Comment:
I think this is not working.

Reply:
Oh! I didn't test this. Let me try this.